src/components/shared/SampleTooltipContents.js

```diff
@@ -136,8 +136,9 @@ export class SampleTooltipContents extends React.PureComponent<Props> {
       const sampleTime = samples.time[sampleIndex];
       const stackIndex = samples.stack[sampleIndex];
       const hasSamples = samples.length > 0 && stackTable.length > 1;
+      const hasValidStackIndex = (stackIndex !== null) ||  (stackIndex !== undefined);
```

Comment:
`||` needs to be `&&` here. We need to make sure that stack index is not null `AND` not undefined. (also there is an extra space on the right side of the or operator).